packages/ckeditor5-table/src/ui/inserttableview.js

```diff
@@ -119,14 +132,38 @@ export default class InsertTableView extends View {
 		this.on( 'change:rows', () => {
 			this._highlightGridBoxes();
 		} );
+
+		this.focusTracker.on( 'change:focusedElement', ( evt, name, focusedElement ) => {
+			if ( !focusedElement ) {
+				return;
+			}
+			const { row, column } = focusedElement.dataset;
+
+			// As row & column indexes are zero-based transform it to number of selected rows & columns.
+			this.set( {
+				rows: parseInt( row ),
+				columns: parseInt( column )
+			} );
+		} );
+	}
+
+	render() {
+		super.render();
+
+		addKeyboardHandlingForGrid( this.keystrokes, this.focusTracker, this.items, 10 );
+
+		for ( const item of this.items ) {
+			this.focusTracker.add( item.element );
+		}
+
+		this.keystrokes.listenTo( this.element );
 	}
 
 	/**
 	 * @inheritDoc
 	 */
 	focus() {
-		// The dropdown panel expects DropdownPanelFocusable interface on views passed to dropdown panel. See #30.
-		// The method should be implemented while working on keyboard support for this view. See #22.
+		this.items.get( 0 ).focus();
```

Comment:
A CSS-related comment but I'm leaving it around here: I noticed the focused grid element has a weird (bold) outline. I think we should get rid of it, the mouse and keyboard selections should look the same IMO.

<img width="210" alt="image" src="https://user-images.githubusercontent.com/1099479/182323739-896f53a8-5879-458d-a774-5f3d5cccacdc.png">